widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -1570,7 +1570,15 @@ button#schedule-dialog-select-button {
   word-break: break-all;
 }
 /* endregion Responsive File Properties dialog - End */
+/* region Select User or Role dialog - Start */
+.select-user-or-role-dialog > div > table {
+  width: 100% !important;
+}
```

Comment:
This rule should target all GWT responsive dialogs. Even if only some dialogs are currently setting the fixed width in this way, it would be best to ensure it is undone for all.

Please, place near the existing rule for `.pentaho-dialog.pentaho-gwt.responsive > * > table`.

Reply:
Addressed
I was not sure how it would affect other dialogs.

I have not added this CSS to an existing block, rather created another block just for this. Should it be in the existing block?

Reply:
And, thanks for changing to `auto`, as well!